dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -495,6 +495,22 @@ function onSkinTemplateNavigationUniversal( SkinTemplate $skin, array &$links )
 wfLoadExtension( 'Score' );
 $wgMusicalNotationEnableWikibaseDataType = true;
 
+# DismissableSiteNotice - https://www.mediawiki.org/wiki/Extension:DismissableSiteNotice
+wfLoadExtension( 'DismissableSiteNotice' );
+$wgMajorSiteNoticeID = 1;
+$wgDismissableSiteNoticeForAnons = true;
```

Comment:
Should this be moved into the `if` condition below as well? Or does it cause build problems?

Reply:
It does not cause build problems I think. Not sure what I prefer, probably a good idea to only load what we need though